### PR TITLE
Adding last mod date for targets

### PR DESCRIPTION
**Pull request by suvamM**
Adding last modified date for scan targets. Tested via App Insights.

### CODE REVIEW

File: src/BinSkim.Sdk/CompilerDataLogger.cs [hunk 1]
```diff
@@ -185,7 +185,8 @@ private void PrintHeader()
             string header = "" +
                 "Target,Compiler Name,Compiler BackEnd Version,Compiler FrontEnd Version," +
                 "File Version,Binary Type,Language,Debugging FileName,Debugging FileGuid," +
-                "Debugging FileLastModifiedDateUTC, Command Line,Dialect,Module Name,Module Library,Hash,Error";
+                "Debugging FileLastModifiedDateUTC, Target LastModifiedDateUTC, Command Line," +
+                "Dialect,Module Name,Module,Library,Hash,Error";
```
Comment (shaopeng-gh) [inline src/BinSkim.Sdk/CompilerDataLogger.cs:189]: >Module,Library [](http://example.com/codeflow?start=37&length=14)

extra , 
:)

**Review by shaopeng-gh — APPROVED** [review]
:shipit:

File: src/BinSkim.Sdk/CompilerDataLogger.cs (hunk 1 above)
Comment (suvamM) [reply src/BinSkim.Sdk/CompilerDataLogger.cs]: Nice catch! Fixed.

**Review by shaopeng-gh — CHANGES_REQUESTED** [review]
:clock1:

**Review by shaopeng-gh — APPROVED** [review]
:shipit:

**Pull request merged**